R/AllGenerics.R

```diff
@@ -0,0 +1,104 @@
+#' @include hidden_aliases.R
+NULL
+
+#' @rdname hidden_aliases
+setGeneric("backendInitialize",
+           def = function(object, ...)
+               standardGeneric("backendInitialize"),
+           valueClass = "ChromBackend"
+           )
+#' @rdname hidden_aliases
+setGeneric("backendMerge", def = function(object, ...)
+    standardGeneric("backendMerge"),
+    valueClass = "ChromBackend")
+#' @rdname hidden_aliases
+setGeneric("chromData", function(object, ...) standardGeneric("chromData"))
+#' @rdname hidden_aliases
+setGeneric("chromData<-", function(object, value)
+    standardGeneric("chromData<-"))
+#' @rdname hidden_aliases
+setGeneric("chromNames", function(object, ...) standardGeneric("chromNames"))
+#' @rdname hidden_aliases
+setGeneric("chromNames<-", function(object, value)
+    standardGeneric("chromNames<-"))
+#' @rdname hidden_aliases
+setGeneric("chromVariables", function(object, ...)
+    standardGeneric("chromVariables"))
+#' @rdname hidden_aliases
+setGeneric("dataOrigin", function(object, ...) standardGeneric("dataOrigin"))
+#' @rdname hidden_aliases
+setGeneric("dataOrigin<-", function(object, value)
```

Comment:
This and other generics are now in `ProtGenerics` 1.17.4.

Reply:
Yep. I will update that (also in `Spectra`)